src/main/java/net/sf/jabref/logic/formatter/bibtexfields/NormalizeNamesFormatter.java

```diff
@@ -36,6 +44,75 @@ public String getKey() {
 
     @Override
     public String format(String value) {
+        Objects.requireNonNull(value);
```

Comment:
Does it really make sense to use this in `AuthorList`? Isn't `NormalizeNameFormatter` quite a bit about guessing? I'd prefer if `AuthorList` didn't guess.